### ISSUE TITLE
Is it possible to integrate Sakuli with SahiPro 7.5.0?

### ISSUE THREAD

**Issue by rahulmr** [ConSol/sakuli#338]
In our org, we already have paid version of Sahi that is Sahi Pro, we are able to utilize the Desktop add-on of SahiPro but many elements of our application cannot be recognized by Sahi Desktop add-on and we are already using Sikuli to do the same. 
We want to embed Sakuli with SahiPro, is there any possible way or guideline to do so?

Actually, we have web automation scripts written in SahiPro, these scripts do not work on SahiOS, and re-work will take a lot of time.

Most of our scenario are Web + Win, so the integration will be very useful.

Thanks in advance for pointing me out to right direction.

**Comment by rahulmr** [comment]
Any updates on this guys?

**Comment by s1hofmann** [comment]
> We want to embed Sakuli with SahiPro

Are you trying to switch Sahi OS for Sahi Pro in Sakuli?
Theoretically, it should be possible to switch Sahi OS for Sahi Pro in Sakuli, but unfortunately I can't provide you with any details or hints since we do not own a license for Sahi Pro.

**Comment by rahulmr** [comment]
@s1hofmann thanks for replying. Just wanted to know whether you modify any files in Sahi OS and which version of Sahi OS are you using? Is it the latest one that is available on SF v5.1? What about upgrading the SikuliXapi jar to the latest version which was released very recently? Also, it is possible to get a trial license of Sahi Pro for one month and there is a Sahi Starter version which is provided by Sahi Team and has a license of 180 days. I would be happy to share a docker image with Sahi Pro starter image with 180 days license.

**Comment by s1hofmann** [comment]
Hi @rahulmr,

in theory it should be possible to substitute Sahi OS for Sahi Pro. I'm saying in theory, because Sakuli hooks onto the Sahi API and I do not know whether there has anything changed from Sahi OS v5.1 (which we're using in Sakuli) and the latest Sahi Pro version.
We're aware of the fact that there's also a trial version for Sahi Pro, but to be honest, we're not too happy with relying on a paid license of Sahi.

So to sum things up: You could give it a try to substitute Sahi OS with Sahi Pro, which should also be possible with our provided Docker Images, but we're not putting any effort into this topic at the moment. If it works out for you, we'd be happy to hear from you!

**Comment by rahulmr** [comment]
@s1hofmann thanks for your reply. I will try and let you know.
